app/models/reading.rb

```diff
@@ -21,8 +21,12 @@ def self.mark_to_read_later(work, user)
 
   # called from rake
   def self.update_or_create_in_database
-    REDIS_GENERAL.smembers("Reading:new").reverse.each do |reading_json|
-      Reading.reading_object(reading_json)
+    REDIS_GENERAL.smembers("Reading:new").reverse.each_slice(ArchiveConfig.READING_BATCHSIZE || 1000) do |batch|
```

Comment:
The plan would be to use the fall back of 1000. I have started using that pattern for small bits of config.